lib/elixir/lib/macro.ex

```diff
@@ -370,6 +370,25 @@ defmodule Macro do
       iex> Macro.escape({:unquote, [], [1]}, unquote: true)
       1
 
+  The `Macro.escape/2` function is sometimes confused with `Kernel.quote/2`,
```

Comment:
You don't need to use the module name when the function you're referring to is in the current module. Please use `escape/2` here and in other places instead of `Macro.escape/2`.

Reply:
Also, please move this paragraph to the following section "Comparison to Kernel.quote/2".